Client/src/RTypeInputListener.cpp

```diff
@@ -0,0 +1,61 @@
+//
+// Created by pasteu_e on 27/11/16.
+//
+
+#include <iostream>
+#include "RTypeInputListener.hpp"
+
+//TODO : Remove all output and send event to NetworkClient by eventEmitter
```

Comment:
EventEmitter is in shared, you might wanna take this out quickly before we forget (it will require you a few changes)